R/dummy_functions.R

```diff
@@ -18,7 +18,7 @@ example_module <- function(label = "example teal module", datanames = "all") {
   ans <- module(
     label,
     server = function(id, data) {
-      checkmate::assert_class(data(), "teal_data")
+      checkmate::assert_class(isolate(data()), "teal_data")
```

Comment:
before calling reactive - first assert whether `data` is reactive

```suggestion
      checkmate::assert_class(data, "reactive")
      checkmate::assert_class(isolate(data()), "teal_data")
```

Reply:
if it's keeps repeating we might consider an util function `assert_reactive()` and `assert_teal_data()` which will call `assert_reactive()` inside
so this essentially could be one liner

Reply:
I think it does keep repeating